RecordingApp/app/src/main/AndroidManifest.xml

```diff
@@ -48,11 +48,11 @@
             android:theme="@style/TR"
             />
 
-        <activity android:name=".ProjectManager.ActivityProjectManager"
+        <activity android:name=".ProjectManager.activities.ActivityProjectManager"
```

Comment:
I see what you're saying. It could be confusing.

Since you said that project manager is an abstract concept, perhaps in the future we could re-arrange these classes into more of a concrete structure. For example: `wycliffeassociates.recordingapp.PlaybackActivity` that imports from `wycliffeassociates.recordingapp.visualization.WavPlayer`,
`wycliffeassociates.recordingapp.entities.Book`, and `wycliffeassociates.recordingapp.widget.SourceAudioPlayer`.